packages/runtime/src/templates/getHandler.ts

```diff
@@ -130,6 +130,15 @@ const makeHandler = (conf: NextConfig, app, pageRoot, staticManifest: Array<[str
     console.log(
       `[${event.httpMethod}] ${event.path} (${requestMode?.toUpperCase()}${result.ttl > 0 ? ` ${result.ttl}s` : ''})`,
     )
+
+    const graphToken = event.netlifyGraphToken
```

Comment:
At this point the request will already have been processed, so this will only set it for the _next_ request

Reply:
Thanks!